rest/model/patch.go

```diff
@@ -189,6 +190,18 @@ func (apiPatch *APIPatch) BuildFromService(h interface{}) error {
 	apiPatch.DownstreamTasks = downstreamTasks
 	apiPatch.ChildPatches = childPatches
 
+	// set the patch status to the collective status between the parent and child patches
+	if len(childPatches) > 0 {
+		allPatches := []APIPatch{}
+		allPatches = append(allPatches, *apiPatch)
+		for _, cp := range childPatches {
+			allPatches = append(allPatches, cp)
```

Comment:
can do `allPatches = append(allPatches, childPatches..)` instead of the loop

Reply:
ah good point. There was more logic in there before, but this loop is no longer necessary. Thanks!